source/docs/advanced/features/password-fields.md

```diff
@@ -0,0 +1,66 @@
+---
+id: password-fields
+title: Password fields
+---
+
+
+# Configure password validity
+
+<blockquote class="feature--new">
+_Since version 2.12.0
+</blockquote>
+
+Front-Commerce expects a certain level of complexity for the password entered by a user.
+The default is:
+- 8 characters minimum
+- at least 3 types of characters among: lowercase letters, uppercase letters, digits and special characters
+
+You can customize those default rules by overriding `theme/components/atoms/Form/Input/Password/passwordConfig.js`
+
+The configuration expects a list of rules to be satisfied and status to be displayed to users. It should follow the format below:
+
+```javascript
+// theme/components/atoms/Form/Input/Password/passwordConfig.js
+import { defineMessages } from "react-intl";
+
+const messages = defineMessages({
+  invalid: {
+    id: "components.atoms.Form.Input.PasswordStrengthHint.Status.invalid",
+    defaultMessage: "This is an invalid password",
+  },
+});
+
+export default {
+  rules: [
+    {
+      // technical ID, must be unique
+      id: "a-unique-technical-id", 
+       // optional, enforces this status if isValid returns false
+      invalidStatus: "TOO_SHORT",
+      // optional, label to display for this rule in the hinter. If it is not defined, the rule is not displayed.
+      label: messages.invalid, 
+      // validation method for this rule
+      isValid: (password) => true, 
+    },
+  ],
+  status: {
+    // status key, used by rule.invalidStatus
+    TOO_SHORT: { 
+      // message to display for this status
+      label: messages.?,
```

Comment:
issue: there is another message to add here (needs to be added to the `messages` list above):

```suggestion
      label: messages.tooShort,
```